src/index.ts

```diff
@@ -88,7 +88,15 @@ export class Transport {
 	/**
 	 * Closes the connection.
 	 */
-	public async close() {
+	public async close(flush: boolean = false) {
```

Comment:
As discussed, please change this to an optional options object so that additional options can be added later without introducing breaking changes.
```suggestion
	public async close(options?: {flush: boolean = false}) {
```